frontend/src/interfaces/ObjectTag.ts

```diff
@@ -0,0 +1,6 @@
+import type { Tag } from '@/interfaces';
+
+export interface ObjectTag {
```

Comment:
Should this be renamed to TagSet? It's name currently implies a singular tag.

Reply:
Might be wrong but as it contains objectId and tagSet, it seems like ObjectTag is more appropriate. @jujaga @TimCsaky ?

Reply:
agree.
it looks good now